src/views/ThresholdUpdates.vue

```diff
@@ -521,6 +527,14 @@ export default defineComponent({
         this.isJobDetailAnimationCompleted = true;
       }
     },
+    async getProductCount(query: any){
```

Comment:
I think we should do it in Job Configuration component and also we should get only count.